### PR TITLE
fix: support of snyk-to-html for unmanaged c/c++

### PR DESCRIPTION
**Pull request by dagrest**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Commits are squashed and tidy and are suitable to become release notes

### What this does

Added support of snyk-to-html for Unmanaged (C/C++)

### Notes for the reviewer

snyk test --unmanaged --json | <path to snyk-to-html project>/dist/index.js -o result.html

### More information

- [Snyk CI plugins support for unmanaged TUN27](https://snyksec.atlassian.net/browse/TUN-27)

### CODE REVIEW

File: src/lib/snyk-to-html.ts [hunk 1]
```diff
@@ -203,6 +203,9 @@ async function generateTemplate(data: any,
   data.uniqueCount = vulnMetadata.vulnerabilitiesUniqueCount;
   data.summary = vulnMetadata.vulnerabilitiesPathsCount + ' vulnerable dependency paths';
   data.showSummaryOnly = summary;
+  if(data.paths && data.paths.length == 1){
+    data.packageManager = data.paths[0].packageManager;
+  }
```
Comment (danielroymoore) [inline src/lib/snyk-to-html.ts:208]: Would be better to write as `data.paths?.length === 1`

File: template/test-report.vuln-card.hbs [hunk 2]
```diff
@@ -26,6 +26,7 @@
                 {{/if_eq}}
             </li>
 
+            {{#if_not_eq list.[0].packageManager "Unmanaged (C/C++)"}}
```
Comment (danielroymoore) [inline template/test-report.vuln-card.hbs:29]: Is it safe to access the list directly like this?

Comment (dagrest) [reply template/test-report.vuln-card.hbs:29]: I guess yes - I saw a lot of usage in the code...

File: src/lib/snyk-to-html.ts (hunk 1 above)
Comment (dagrest) [reply src/lib/snyk-to-html.ts:208]: Yes, it will be good!

**Review by danielroymoore — APPROVED** [review]

**Comment by PeterSchafer** [comment]
Hi @dagrest, would it be possible to add or extend some of the tests to incorporate the changes in logic?

**Comment by dagrest** [comment]
> Hi @dagrest, would it be possible to add or extend some of the tests to incorporate the changes in logic?

Sure, will do that!

**Review by PeterSchafer — APPROVED** [review]

**Pull request merged**